roles/ironic-control/meta/main.yml

```diff
@@ -9,4 +9,3 @@ dependencies:
     db_version: 'db versions'
     db_sync_cmd: 'ironic-dbsync'
     db_sync: 'upgrade'
-  - role: openstack-source
```

Comment:
This is being removed here, but I don't see it being added to the ironic-common dependencies. Was that missed (since it would be a new file)?